<launch>
  <arg name="car" default="true"/>
  <arg name="pedestrian" default="false"/>

  <arg name="camera_id" default="/"/>

  <group if="$(arg car)">
    <group ns="obj_car">

      <arg name="image_src_car" default="/image_raw"/>

      <node pkg="cv_tracker" name="tracking" type="kf_track" output="screen">
          <remap from="/config/obj_car/kf" to="/config/car_kf" />
          <param name="image_node" type="str" value="$(arg camera_id)$(arg image_src_car)"/>
      </node>

    </group>
  </group>

  <group if="$(arg pedestrian)">
    <group ns="obj_person">

      <arg name="image_src_pedestrian" default="/image_raw"/>

      <node pkg="cv_tracker" name="tracking" type="kf_track">
          <remap from="/config/obj_person/kf" to="/config/pedestrian_kf" />
          <param name="image_node" type="str" value="$(camera_id)$(arg image_src_pedestrian)"/>
      </node>

    </group>
  </group>

</launch>
